generators/rails-generator/tasks/action_pack.rake

```diff
@@ -0,0 +1,144 @@
+stdlib_dependencies = %w[benchmark date digest json logger monitor mutex_m pathname singleton time minitest securerandom ipaddr did_you_mean]
+gem_dependencies = %w[nokogiri i18n rack rails-dom-testing]
+rails_dependencies = %w[activesupport actionview]
+
+VERSIONS.each do |version|
+  namespace version do
+    namespace :action_pack do
+      export = "export/actionpack/#{version}"
+
+      desc "export to #{export}"
+      task :export do
+        sh "rm -fr #{export}"
+        sh "mkdir -p #{export}"
+
+        sh "cp -a out/#{version}/action_pack.rbs #{export}"
+        sh "cp -a out/#{version}/action_pack #{export}"
+        sh "cp -a out/#{version}/action_controller.rbs #{export}"
+        sh "cp -a out/#{version}/action_controller #{export}"
+        sh "cp -a out/#{version}/action_dispatch.rbs #{export}"
+        sh "cp -a out/#{version}/action_dispatch #{export}"
+        sh "cp -a out/#{version}/abstract_controller.rbs #{export}"
+        sh "cp -a out/#{version}/abstract_controller #{export}"
+        sh "cp -a out/#{version}/mime.rbs #{export}"
+        sh "cp -a out/#{version}/mime #{export}"
+
+        sh "rm #{export}/action_controller/railtie.rbs"
+        sh "rm #{export}/action_dispatch/railtie.rbs"
+
+        Pathname(export).join("EXTERNAL_TODO.rbs").write(<<~RBS)
```

Comment:
Why the files are written in the `.rake` file? I think it is better if this file is placed as a `.rbs` file. It improves editor integrations and so on.